sky/engine/core/script/dart_service_isolate.h

```diff
@@ -37,6 +39,8 @@ class DartServiceIsolate {
   // Observatory resource loading.
   static Dart_Handle LoadResources(Dart_Handle library);
   static Dart_Handle LoadResource(Dart_Handle library, const char* name);
+
+  static int observatory_port_;
```

Comment:
You could also make this a file-level static, which would make the symbol private.